controller/src/test/java/io/pravega/controller/task/Stream/StreamMetadataTasksTest.java

```diff
@@ -133,20 +134,29 @@ public void tearDown() throws Exception {
         executor.shutdown();
     }
 
-    @Test
+    @Test(timeout = 10000)
     public void sealStreamTest() throws Exception {
-        assertNotEquals(0, consumer.getCurrentSegments(SCOPE, stream1).get().size());
+        List<Controller.SegmentRange> segmentRanges = consumer.getCurrentSegments(SCOPE, stream1).get();
+        assertNotEquals(0, segmentRanges.size());
 
         //seal a stream.
-        UpdateStreamStatus.Status sealOperationResult = streamMetadataTasks.sealStreamBody(SCOPE, stream1, null).get();
-        assertEquals(UpdateStreamStatus.Status.SUCCESS, sealOperationResult);
+        ScaleResponse scaleResponse = streamMetadataTasks.manualScale(SCOPE, stream1,
```

Comment:
thats because you also need to update asserts in streamMetadataTasks that expect 'new segments created' to be non-empty.
change those asserts to be newSegmentsCreated.size() == input.newRanges.size()
(there are two of them)
also update the scale timestamp which is taken as newSegment.creationTime and you can use System.currentTimeInMillis there.